frontend/packages/ceph-storage-plugin/src/components/modals/storage-pool-modal/storage-pool-modal.tsx

```diff
@@ -87,6 +91,22 @@ export const StoragePoolModal = withHandlePromise((props: StoragePoolModalProps)
   const [isSubmit, setIsSubmit] = React.useState(false);
   const [timer, setTimer] = React.useState<NodeJS.Timer>(null);
   const [isArbiterCluster, setArbiterCluster] = React.useState(false);
+  const [pools, poolsLoaded, poolsLoadedError] = useK8sGet<ListKind<StoragePoolKind>>(
+    CephBlockPoolModel,
+    null,
+    CEPH_STORAGE_NAMESPACE,
```

Comment:
You can fetch the specific default pool by name as well, instead of getting list
```suggestion
    CephBlockPoolModel,
    OCS_INTERNAL_CR_NAME,
    CEPH_STORAGE_NAMESPACE,
```

Reply:
The only concern is that name should not be different for 4.6 and 4.8 alongwith 4.7.
If its same for all above , then we an use, otherwise existing list fetching is okay.

Reply:
Finding a default pool using a name may cause an error in the future, this approach is safe. only the storage cluster is going to own the default pool.